app/src/main/res/values/strings.xml

```diff
@@ -9,6 +9,7 @@
     <string name="cancel">Cancel</string>
     <string name="fdroid_vlc_url" translatable="false">https://f-droid.org/repository/browse/?fdfilter=vlc&amp;fdid=org.videolan.vlc</string>
     <string name="open_in_browser">Open in browser</string>
+    <string name="get_url">Get URL</string>
```

Comment:
I'd use "Copy URL" instead of "get". Also change this in code

Reply:
Ok, I was thinking about copy or get, but I choose get, since get is more like: get the url of this video. But it makes sense to be copy as well.